Sources/CreateOptions/ConfigOptions.swift

```diff
@@ -568,6 +568,18 @@ public struct ConfigOptions: ParsableConfiguration {
         @Option public var filenameTemplate: String = "%0.swift"
     }
 
+    @Option public var enums: Enums
+
+    // sourcery: document,
+    /// Options specifically related to generating enums of strings.
+    ///
+    /// Note: these options do not apply to other enums, such as `CodingKey` conformances or
```

Comment:
Just a suggestion to get the nicer formatting in GitHub

```suggestion
    /// > **Note**: these options do not apply to other enums, such as `CodingKey` conformances or
```

It would look like this:

> **Note**: these options do not apply to other enums, such as `CodingKey` conformances or `ResponseHeaders`.